packages/test/src/helpers-integration.ts

```diff
@@ -47,36 +50,50 @@ const defaultDynamicConfigOptions = [
 
 export function makeTestFunction(opts: {
   workflowsPath: string;
-  workflowEnvironmentOpts?: LocalTestWorkflowEnvironmentOptions;
+  localWorkflowEnvironmentOpts?: LocalTestWorkflowEnvironmentOptions;
+  timeSkippingWorkflowEnvironmentOpts?: LocalTestWorkflowEnvironmentOptions;
   workflowInterceptorModules?: string[];
 }): TestFn<Context> {
   const test = anyTest as TestFn<Context>;
   test.before(async (t) => {
     // Ignore invalid log levels
     Runtime.install({ logger: new DefaultLogger((process.env.TEST_LOG_LEVEL || 'DEBUG').toUpperCase() as LogLevel) });
-    const env = await TestWorkflowEnvironment.createLocal({
-      ...opts.workflowEnvironmentOpts,
+    const envLocal = await TestWorkflowEnvironment.createLocal({
+      ...opts.localWorkflowEnvironmentOpts,
       server: {
-        ...opts.workflowEnvironmentOpts?.server,
+        ...opts.localWorkflowEnvironmentOpts?.server,
         extraArgs: [
           ...defaultDynamicConfigOptions.flatMap((opt) => ['--dynamic-config-value', opt]),
-          ...(opts.workflowEnvironmentOpts?.server?.extraArgs ?? []),
+          ...(opts.localWorkflowEnvironmentOpts?.server?.extraArgs ?? []),
         ],
       },
     });
-    await registerDefaultCustomSearchAttributes(env.connection);
+    await registerDefaultCustomSearchAttributes(envLocal.connection);
+    const envTimeSkipping = RUN_TIME_SKIPPING_TESTS
+      ? await TestWorkflowEnvironment.createTimeSkipping({
+          ...opts.timeSkippingWorkflowEnvironmentOpts,
+          server: {
+            ...opts.timeSkippingWorkflowEnvironmentOpts?.server,
+            extraArgs: [...(opts.timeSkippingWorkflowEnvironmentOpts?.server?.extraArgs ?? [])],
+          },
+        })
+      : undefined;
     const workflowBundle = await bundleWorkflowCode({
       ...bundlerOptions,
       workflowInterceptorModules: [...defaultWorkflowInterceptorModules, ...(opts.workflowInterceptorModules ?? [])],
       workflowsPath: opts.workflowsPath,
     });
     t.context = {
-      env,
+      envLocal,
+      envTimeSkipping,
```

Comment:
I was imagining every test run would be parameterized to run either with the CLI or the time skipping server instead of creating both.

Reply:
Yes, but both must be created and terminated from the before/after hooks.